src/installer/install.ts

```diff
@@ -61,23 +61,37 @@ export async function downloadAndInstall(file: ClientFile): Promise<string> {
     let clientExecutableTmpPath;
     if (extractedDir) {
         const executable = getExecutable(file);
-        const execuatbleFileGlob = `${extractedDir}/**/${executable}`;
+
+        // array consisting of possiblilties of executable file name
+        const execuatbleFileGlobArray: string[] = [];
```

Comment:
I notice that all the entries in this array start with the `dirname/**`. So you could remove a bit of duplication here by removing that prefix from each entry. Then the glob array would just contain the file names.

Then, when you do ghGlob.create you can use `Array.map` to add the dirname pattern to every entry. as follows:

```js
ghGlob.create(executableFileGlobArray.map((executableGlob) => `${extractedDir}/**/${executableGlob}).join("\n"))
```

Reply:
Okay, thanks :+1: